symengine/ruby/spec/basic_spec.rb

```diff
@@ -16,56 +16,5 @@
                 end
             end
         end
-
-        describe 'binary operations' do
-            before :each do
-                @a = SymEngine::Basic.new
-                @b = SymEngine::Basic.new
-            end
-            describe '#+' do
-                context 'with another Basic object as argument' do
-                    xit 'returns a Basic object that is result of
-                        self added to the argument' do
```

Comment:
All the operators here are common to all `Basic` classes. So, I think it makes sense to keep them here. You could have `symbols`, that's okay. I would define only `Symbol` specific methods in the other file.